src/notation/view/widgets/editstyle.cpp

```diff
@@ -172,13 +173,13 @@ class EditStyle::LineStyleSelect : public QObject
     QList<QWidget*> dashSpecificWidgets;
 };
 
-static constexpr const char* lineStyles[] = {
+static const TranslatableString lineStyles[] = {
     //: line style
-    QT_TRANSLATE_NOOP("notation/editstyle", "Continuous"),
+    TranslatableString("notation/editstyle", "Continuous"),
     //: line style
-    QT_TRANSLATE_NOOP("notation/editstyle", "Dashed"),
+    TranslatableString("notation/editstyle", "Dashed"),
     //: line style
-    QT_TRANSLATE_NOOP("notation/editstyle", "Dotted"),
+    TranslatableString("notation/editstyle", "Dotted"),
```

Comment:
There are still a few `QT_TRANSLATE_NOOP` elsewhere in the code, is that a problem?

Reply:
Not per se. It becomes a problem at the moment that we forget to sync the context passed to `qtrc` with the context specified in `QT_TRANSLATE_NOOP`.

Reply:
How to tell one from the other?

Reply:
The contexts must simply match. But it may be difficult to track down where a string ends up being used, so if you update the context in the `QT_TRANSLATE_NOOP`, you will have to find out manually where that string can go and where it is translated using `qtr`, and update the context there too.
(Is that what you mean?)

Reply:
Yeah, I guess that's what I meant